org.widelands.widelands.json

```diff
@@ -0,0 +1,478 @@
+                "install -d '/app/opt/imagemagick/share/doc/imagemagick/Magick++/examples'; install -p -m 0644 Magick++/examples/* '/app/opt/imagemagick/share/doc/imagemagick/Magick++/examples/';",
+                "echo 'post-install: licenses';",
+                "install -d '/app/opt/imagemagick/share/licenses/imagemagick'; install -p -m 0644 'LICENSE' '/app/opt/imagemagick/share/licenses/imagemagick/';",
+                "install -d '/app/opt/imagemagick/share/licenses/imagemagick/Magick++'; install -p -m 0644 'www/Magick++/COPYING' '/app/opt/imagemagick/share/licenses/imagemagick/Magick++/';"
+            ],
+            "cleanup": [
+                "/opt/imagemagick/share/doc",
+                "/opt/imagemagick/share/man"
+            ]
+        },
+        "shared-modules/lua5.3/lua-5.3.2.json",
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/glew/glew.json",
+        {
+            "name": "boost",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.bz2",
+                    "sha256": "2684c972994ee57fc5632e03bf044746f6eb45d4920c343937a465fd67a5adba"
+                },
+                {
+                    "type": "script",
+                    "commands": [
+                        "./bootstrap.sh --prefix=/app --with-libraries=filesystem,system,signals,regex,test;"
+                    ],
+                    "dest-filename": "configure"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'all:\n\t./b2 headers;\n\ninstall:\n\t./b2 install;\n' > 'Makefile';"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "widelands",
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_PREFIX=/app/bin",
+                "-DWL_INSTALL_BASEDIR=/app/share/widelands",
+                "-DWL_INSTALL_DATADIR=/app/share/widelands",
+                "-DBOOST_ROOT=/app",
+                "-DGLEW_ROOT=/app",
+                "-DGLEW_GLEW_LIBRARY=/app/lib/libGLEW.so",
+                "-DGLEW_INCLUDE_DIR=/app/include/GL",
+                "-DGLEW_LIBRARY:FILEPATH=/app/lib/libGLEW.a",
+                "-DOPENGL_glu_LIBRARY=/app/lib/libGLU.a",
+                "-DOPTION_BUILD_WEBSITE_TOOLS=OFF"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/widelands/build19/build19/+download/widelands-build19-src-gcc7.tar.bz2",
+                    "sha256": "ef73b8cf5a7c680bad5caade907f1cc6ee88dd3729470b9214a3c72658585458"
+                },
+                {
+                    "type": "file",
```

Comment:
You don't need checksums for local file. There is no re-usable icon in the tarball?

Reply:
> There is no re-usable icon in the tarball?

There was only a low resolution PNG file.